setup.sh

```diff
@@ -67,7 +65,7 @@ header_text "Waiting for istio to become ready"
 sleep 5; while echo && kubectl get pods -n istio-system | grep -v -E "(Running|Completed|STATUS)"; do sleep 5; done
 
 header_text "Setting up Knative Serving"
-curl -L "https://raw.githubusercontent.com/openshift-knative/knative-serving-operator/master/deploy/resources/knative-serving-${serving_version}.yaml" \
+curl -L "https://github.com/knative/serving/releases/download/${serving_version}/serving.yaml" \
   | sed 's/LoadBalancer/NodePort/' \
   | kubectl apply --filename -
 
```

Comment:
I have something like 

```
set +e
kubectl apply --selector knative.dev/crd-install=true --filename https://storage.googleapis.com/knative-nightly/serving/latest/serving.yaml
rc=$?
set -e
if [ $rc -ne 0 ]; then
   # Retry
  sleep 5
  kubectl apply --selector knative.dev/crd-install=true --filename https://storage.googleapis.com/knative-nightly/serving/latest/serving.yaml
fi
```

the URL need to be replaced, but this just retries to apply the CRDs if it fails the first time.